### PR TITLE
List/View Class and Relationships

### PR DESCRIPTION
**Pull request by 16aburnett**
Implementations for listing the attributes of a given class and listing the relationships of all classes

### CODE REVIEW

**Comment by carlishires** [comment]
Since the execute function is not complete is there any way I can test this?

**Comment by 16aburnett** [comment]
> Since the execute function is not complete is there any way I can test this?

Joan and I were testing by calling the method in the REPL function and we manually added data in the model constructor

**Comment by 16aburnett** [comment]
listview

I modified list_class to check if the class name exists. These are the tests i performed and the outputs. 

this is the model's state: 
c1 has attributes [A B C]
c2 has attributes [D E F]
c3 has attributes []
c1 and c2 share a relationship
c1 and c3 share a relationship

File: code/UMLModel.py [hunk 1]
```diff
@@ -88,15 +88,33 @@ def load_model(self):
     ######################################################################
 
     # **Write Documentation Here**
-    def list_class(self):
-        print ("To be implemented")
+    # Output all attributes and methods for a given class
+    def list_class(self, class_name):
```
[the remark this answers is not in the document]
Comment (16aburnett) [reply code/UMLModel.py:92]: That's a great observation. We really should have consistency so I will add that.

File: code/UMLModel.py [hunk 2]
```diff
@@ -88,15 +88,33 @@ def load_model(self):
...
+                print(attribute)
+        # not valid class
+        else:
+            print (f"{class_name} is not a class")
 
     ######################################################################
 
     # **Write Documentation Here**
+    # Output all of the relationships between classes
     def list_relationships(self):
```
[the remark this answers is not in the document]
Comment (16aburnett) [reply code/UMLModel.py:107]: Yeah, that's something I kinda noticed. We probably should have other methods for printing. We also don't have a function that prints all the classes. List_class should probably be called list_attributes.

File: code/UMLModel.py [hunk 3]
```diff
@@ -88,15 +88,33 @@ def load_model(self):
     ######################################################################
 
     # **Write Documentation Here**
-    def list_class(self):
-        print ("To be implemented")
+    # Output all attributes and methods for a given class
```
[the remark this answers is not in the document]
Comment (16aburnett) [reply code/UMLModel.py:91]: Yes. At this point, we aren't storing methods with the UMLClass so this should only say attributes.

File: code/UMLModel.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (16aburnett) [reply code/UMLModel.py:105]: Oh, I get what you mean. I guess it should also display the name of the relationship and in the future we will also have other data associated with relationships like the cardinality, and that information should be displayed as well. I could change it to output the name as well.

**Review by adjusic — COMMENTED** [review]
Looks great!

**Review by adjusic — APPROVED** [review]

**Pull request merged**